src/main/java/mezz/modnametooltip/TooltipEventHandler.java

```diff
@@ -33,6 +38,14 @@ public void onToolTip(ItemTooltipEvent event) {
 	private static String getModName(ItemStack itemStack) {
 		if (!itemStack.isEmpty()) {
 			Item item = itemStack.getItem();
+
+			// Override for Groovyscript
+			if (ModNameTooltip.isGroovyLoaded()){
+				if (Objects.requireNonNull(item.getRegistryName()).getResourceDomain().equals(
+						GroovyScript.getRunConfig().getPackId()))
+					return GroovyScript.getRunConfig().getPackName();
+			}

```

Comment:
I'd like to hide away this logic in a file like `GroovyScriptHack.java` and then call it here like
```suggestion
			if (GroovyScriptHack.test(item)) {
				return GroovyScriptHack.getName(item);
			}

```

Reply:
Ok, I'll do this later today.

Reply:
Done. I've created a folder called compat, in case this happens with another mod, and further compat scripts can go there.
https://github.com/mezz/ModNameTooltip/pull/30/commits/19c35e9e72a8dff48841669ae186f8129a06395a